.env.example

```diff
@@ -1,6 +1,9 @@
 # The port for the development server
 PORT=8080
 
+VUE_APP_WIKIBASE_API_URL=
```

Comment:
If it is an example, then it should have example content :)

Using the same as in .env.production is fine.

Reply:
@micgro42 
In my experience, the env.example files is only supposed to show what env variables there are , and not their actual value.
especially, since its going to be part of the repo (not in .gitignore). 
what if we have secrets in the env file ? we're certainly not going to add those values in the .env.example file. 
this is all just from my experience. I'm happy to hear rebuttals.

Reply:
No, there shouldn't be any secrets in .env file. The point of showing it here is to give an example for what is expected. E.g. that the api url is `https://example.org/w/api.php` and not `https://example.org/w/index.php` or something. I.e. an example file should in principle be valid. I'm not saying that it should be a copy of the production file, just pointing out that what we currently have for those two fields in the production file would be good examples.

You might be thinking of not an `.example` file, but a `.dist` file which is a template or starter that is intended to be copied to `.local`, to be manually adjusted and then read by the application?

Reply:
I see. for now, i just copied over what we have on the env file
i was used to a different workflow. but what you said makes sense, as well